server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/ProcessorUtil.java

```diff
@@ -63,7 +65,7 @@ public static void handleException(Exception me, Mail mail, String offendersName
         String errorString = sout.toString();
         mail.setErrorMessage(errorString);
         logger.error(errorString);
-        mail.setAttribute(Mail.MAILET_ERROR_ATTRIBUTE_NAME, me);
+        mail.setAttribute(new Attribute(Mail.MAILET_ERROR_ATTRIBUTE_NAME, AttributeValue.ofAny(me)));

```

Comment:
I know I'm repeating myself but I'd really like this to be 

`mail.setAttribute(Attribute.of(Mail.MAILET_ERROR_ATTRIBUTE_NAME, me));`